charts/crowdsec/templates/lapi-deployment.yaml

```diff
@@ -72,12 +80,25 @@ spec:
         env:
           - name: MB_DB_FILE
             value: /metabase-data/metabase.db
-      {{ end }}
+      {{- end }}
       terminationGracePeriodSeconds: 30
-      {{ if .Values.lapi.dashboard.enabled }}
+      {{- if or (.Values.lapi.persistentVolume.data.enabled) (.Values.lapi.persistentVolume.config.enabled) (.Values.lapi.dashboard.enabled) }}
```

Comment:
this check is not needed here `(.Values.lapi.persistentVolume.config.enabled)` as the crowdsec config volume isn't used in this container.

Reply:
the crowdsec config file is used in the container crowdsec-lapi or? then we need to load it in the pod or? (We only load it in the crowdsec-lapi container and not in the dashboard, but the volume must be available on pod level)

Reply:
Yes the crowdsec config is only used on the crowdsec-lapi container. You already add it as volume in L66.
VolumesMounts are per container and not pod no ?

Reply:
Yes the crowdsec config is only used on the crowdsec-lapi container. You already add it as volume in L66.
VolumesMounts are per container and not pod no ?

Reply:
I think we have a misunderstanding here. Maybe you have misread the line intends? The current structure looks something like this (deleted a lot of content for simplicity):
```
apiVersion: apps/v1
kind: Deployment
spec:
  template:
    spec:
      initContainers:
        - name: fetch-metabase-config
      containers:
        - name: crowdsec-lapi        
          volumeMounts:
            - name: crowdsec-db
              mountPath: /var/lib/crowdsec/data
            - name: crowdsec-config
              mountPath: /etc/crowdsec
        - name: dashboard
          volumeMounts:
          - name: shared-data
            mountPath: /metabase-data
          - name: crowdsec-db
            mountPath: /var/lib/crowdsec/data
      volumes:
      - name: crowdsec-db
        persistentVolumeClaim:
          claimName: {{ .Release.Name }}-db-pvc
      - name: crowdsec-config
        persistentVolumeClaim:
          claimName: {{ .Release.Name }}-config-pvc
```

We define the pvc in `spec.template.spec.volumes` as a `volume` and give it a name. At this moment the pvc is defined but it is not visible to any container. Then we have two containers here, the metabase dashboard and the lapi container, but only the lapi container can access the volume as the `spec.template.spec.containers.[crowdsec-lapi].volumeMounts` contains a `volumeMount` for the defined volume `crowdsec-config`. The dashboard does not have it and so it can not access the storage.

Reply:
Ah sorry for the misunderstanding. You are actually right :+1:.